vocabulary-draft/continuous-delivery.md

```diff
@@ -0,0 +1,28 @@
+# Continuous Delivery
+
+__Note:__ This is a work-in-progress draft version and is being worked on by members of the Events SIG. You are very welcome to join the work and the discussions!
+
+Continuous Delivery related to activities and orchestration that needs to exist to be able to deterministically and continously being able to delivery software to users. 
+
+A pipeline, in the context of Continuous Delivery, is the definition of a set of tasks that needs to be performed to build, test, package, release and deploy software artifacts. A pipeline can be instanciated multiple times, for example to build different versions of the same artifact. We are refering to this instance as PipelineRun. It will have a unique Id and it will help us to track the build and release progress on a particular software artifact. 
```

Comment:
How do we want to do? Should we state a term and define it or should we also refer to other terminologies. In https://github.com/cdfoundation/sig-interoperability/blob/master/docs/vocabulary.md#mapping-of-terms you can see that there are a lot of tools that speak of a workflow instead.